components/esp_modem/test/host_test/env.sh

```diff
@@ -1,10 +1,20 @@
 #!/bin/bash
 
-lwip=$1
-lwip_uri=$2
-lwip_contrib=$3
-
-wget --no-verbose ${lwip_uri}/${lwip}.zip
-unzip -oq ${lwip}.zip
-wget --no-verbose ${lwip_uri}/${lwip_contrib}.zip
-unzip -oq ${lwip_contrib}.zip
+idf_version=$1
+component=$2
+
+if [ "$idf_version" == "release-v4.3" ] && [ "$component" == "esp_modem" ]; then
+  lwip=lwip-2.1.2
+  lwip_uri=http://download.savannah.nongnu.org/releases/lwip
+  lwip_contrib=contrib-2.1.0
+
+  wget --no-verbose ${lwip_uri}/${lwip}.zip
+  unzip -oq ${lwip}.zip
+  wget --no-verbose ${lwip_uri}/${lwip_contrib}.zip
+  unzip -oq ${lwip_contrib}.zip
+
+  apt-get update && apt-get install -y gcc-8 g++-8
+  update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8
```

Comment:
What is the reason for installing `gcc-8` here? Why we can't use the default one from the image?

Reply:
https://github.com/espressif/esp-protocols/pull/397#discussion_r1380291725